README.md

```diff
@@ -1922,10 +1932,101 @@ scheme: bearer
 bearerFormat: JWT
 ```
 
+###### Implicit OAuth2 Sample
 
+```json
+{
+  "type": "oauth2",
+  "flows": {
+    "implicit": {
+      "authorizationUrl": "https://example.com/api/oauth/dialog",
+      "scopes": {
+        "write:pets": "modify pets in your account",
+        "read:pets": "read your pets"
+      }
+    }
+  }
+}
+```
 
+```yaml
+type: oauth2
+flows: 
+  implicit:
+    authorizationUrl: https://example.com/api/oauth/dialog
+    scopes:
+      write:pets: modify pets in your account
+      read:pets: read your pets
+```
 
+#### <a name="oauthFlowsObject"></a>OAuth Flows Object
 
+Allows configuration of the supported OAuth Flows.
+
+##### Fixed Fields
+Field Name | Type | Description
+---|:---:|---
+<a name="oauthFlowsImplicit"></a>implicit| [OAuth Flow Object](#oauthFlowObject) | Configuration for the OAuth Implicit flow 
```

Comment:
A meaningless comment I know, but should you not be consistent with spacing i.e. 
```
<a name="oauthFlowsImplicit"></a>implicit | [OAuth Flow Object](#oauthFlowObject) | Configuration for the OAuth Implicit flow 
```
if you want to be consistent with the table heading columns and most other columns (apart from `---|:---:|---`)

Reply:
👍 actually a cut-and-paste from the OpenAPI spec - but I'll check for it in future